torchci/lib/fetchHud.ts

```diff
@@ -8,10 +8,10 @@ export default async function fetchHud(params: HudParams): Promise<{
 }> {
   const rocksetClient = getRocksetClient();
   const [hudQuery, commitQuery] = await Promise.all([
-    rocksetClient.queryLambdas.executeQueryLambdaByTag(
+    rocksetClient.queryLambdas.executeQueryLambda(
       "commons",
       "hud_query",
-      "prod",
+      "e7597aba6bba4320",
```

Comment:
So instead of getting the query by tag, we are now switching to getting the query by the version--why is this better?

Reply:
it makes the upgrade atomic without breaking older instances of HUD—so basically, the newly deployed version will use this query, but the old version will use the old query. Otherwise, when we upgraded the prod tag there would be a temporary outage before we pushed the new version of torchci; not a big deal but since it's avoidable, might as well avoid it.

Reply:
Oh, I understand, thanks!